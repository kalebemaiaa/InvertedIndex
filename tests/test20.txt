QueryAndS(QueryOr(QueryWord("governador"), QueryNot(QueryOr(QueryWord("deputado"), QueryWord("senador")))), QueryWord("prefeito"))
1476.raw
2903.raw
2067.raw
188.raw
1112.raw
3967.raw
915.raw
4228.raw
6371.raw
5294.raw
811.raw
11979.raw
4486.raw
4878.raw
3003.raw
3960.raw
3511.raw
11016.raw
2392.raw
8887.raw
4614.raw
4822.raw
2665.raw
5675.raw
3726.raw
2201.raw
5006.raw
2924.raw
2865.raw
11784.raw
2062.raw
2629.raw
12045.raw
5440.raw
3827.raw
3348.raw
1625.raw
6147.raw
1508.raw
2069.raw
6207.raw
509.raw
11746.raw
5825.raw
3270.raw
837.raw
11096.raw
4008.raw
2898.raw
2926.raw
3971.raw
3769.raw
11669.raw
2718.raw
1639.raw
1729.raw
1344.raw
2287.raw
464.raw
11253.raw
8739.raw
3171.raw
6130.raw
259.raw
11355.raw
381.raw
5215.raw
60.raw
1248.raw
11050.raw
1829.raw
2634.raw
484.raw
4328.raw
4312.raw
10962.raw
6326.raw
11207.raw
1230.raw
5817.raw
5494.raw
4199.raw
5213.raw
4823.raw
3983.raw
1446.raw
3391.raw
1386.raw
6296.raw
7796.raw
2307.raw
2087.raw
978.raw
793.raw
10781.raw
1407.raw
8853.raw
1244.raw
5341.raw
11540.raw
5381.raw
544.raw
3388.raw
835.raw
1637.raw
5000.raw
4324.raw
5061.raw
1353.raw
278.raw
840.raw
1611.raw
3932.raw
2222.raw
5598.raw
1938.raw
2661.raw
3294.raw
3796.raw
4655.raw
2501.raw
2125.raw
1778.raw
2504.raw
3639.raw
6394.raw
3523.raw
3292.raw
11401.raw
5803.raw
520.raw
3318.raw
6077.raw
2212.raw
834.raw
1343.raw
2587.raw
2159.raw
2906.raw
11007.raw
4009.raw
6256.raw
1478.raw
534.raw
11759.raw
10788.raw
1772.raw
3941.raw
8835.raw
4172.raw
4134.raw
5182.raw
2933.raw
2028.raw
11807.raw
1290.raw
2284.raw
1542.raw
2655.raw
4062.raw
6028.raw
1660.raw
3539.raw
3921.raw
4699.raw
8900.raw
4922.raw
2341.raw
1747.raw
5506.raw
5942.raw
3459.raw
5208.raw
4601.raw
5327.raw
3473.raw
3254.raw
11927.raw
5276.raw
5746.raw
245.raw
11934.raw
11713.raw
5184.raw
3891.raw
3457.raw
12046.raw
854.raw
655.raw
3859.raw
4602.raw
98.raw
11103.raw
3722.raw
10920.raw
6071.raw
6128.raw
11218.raw
9.raw
5916.raw
