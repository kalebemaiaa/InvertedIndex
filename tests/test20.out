9
520
544
793
811
837
854
978
1112
1353
1386
1407
1625
1639
1778
2062
2125
2201
2212
2587
2655
2665
3003
3270
3318
3388
3391
3722
3726
3796
3891
3921
4228
4602
4922
5000
5208
5381
5598
10788
10962
11007
11050
11096
11218
11713
11934
